I re-triggered Travis, and now the build fails:


```
Errors << ros_ign_bridge:make /root/catkin_ws/logs/ros_ign_bridge/build.make.000.log

/root/catkin_ws/src/code/ros_ign_bridge/src/convert.cpp: In function 'void ros_ign_bridge::convert_ros_to_ign(const ROS_T&, IGN_T&) [with ROS_T = visualization_msgs::Marker_<std::allocator<void> >; IGN_T = ignition::msgs::Marker]':

/root/catkin_ws/src/code/ros_ign_bridge/src/convert.cpp:1299:48: error: 'ARROW' is not a member of 'ignition::msgs::Marker'

       ign_msg.set_type(ignition::msgs::Marker::ARROW);

                                                ^~~~~

/root/catkin_ws/src/code/ros_ign_bridge/src/convert.cpp: In function 'void ros_ign_bridge::convert_ign_to_ros(const IGN_T&, ROS_T&) [with ROS_T = visualization_msgs::Marker_<std::allocator<void> >; IGN_T = ignition::msgs::Marker]':

/root/catkin_ws/src/code/ros_ign_bridge/src/convert.cpp:1438:34: error: 'CONE' is not a member of 'ignition::msgs::Marker'

     case ignition::msgs::Marker::CONE:

                                  ^~~~

/root/catkin_ws/src/code/ros_ign_bridge/src/convert.cpp:1442:34: error: 'ARROW' is not a member of 'ignition::msgs::Marker'

     case ignition::msgs::Marker::ARROW:

                                  ^~~~~

/root/catkin_ws/src/code/ros_ign_bridge/src/convert.cpp:1445:34: error: 'AXIS' is not a member of 'ignition::msgs::Marker'

     case ignition::msgs::Marker::AXIS:

                                  ^~~~
```

--- a/ros_ign_bridge/CMakeLists.txt
+++ b/ros_ign_bridge/CMakeLists.txt
@@ -29,6 +29,8 @@ if ("$ENV{IGNITION_VERSION}" STREQUAL "blueprint")
   set(IGN_MSGS_VER ${ignition-msgs4_VERSION_MAJOR})
 
   message(STATUS "Compiling against Ignition Blueprint")
+  
+  add_definitions(-DIGNITION_BLUEPRINT)
 elseif("$ENV{IGNITION_VERSION}" STREQUAL "citadel")
   find_package(ignition-transport8 REQUIRED)
   set(IGN_TRANSPORT_VER ${ignition-transport8_VERSION_MAJOR})
@@ -37,6 +39,7 @@ elseif("$ENV{IGNITION_VERSION}" STREQUAL "citadel")
   set(IGN_MSGS_VER ${ignition-msgs5_VERSION_MAJOR})
 
   message(STATUS "Compiling against Ignition Citadel")
+  add_definitions(-DIGNITION_CITADEL)
 else()
   find_package(ignition-transport9 REQUIRED)
   set(IGN_TRANSPORT_VER ${ignition-transport9_VERSION_MAJOR})
@@ -45,6 +48,7 @@ else()
   set(IGN_MSGS_VER ${ignition-msgs6_VERSION_MAJOR})
 
   message(STATUS "Compiling against Ignition Dome")
+  add_definitions(-DIGNITION_DOME)
 endif()
 
 catkin_package()

--- a/ros_ign_bridge/src/convert.cpp
+++ b/ros_ign_bridge/src/convert.cpp
@@ -1295,9 +1295,11 @@ convert_ros_to_ign(
   // Type
   switch(ros_msg.type)
   {
+#ifdef IGNITION_DOME
     case visualization_msgs::Marker::ARROW:
       ign_msg.set_type(ignition::msgs::Marker::ARROW);
       break;
+#endif
     case visualization_msgs::Marker::CUBE:
       ign_msg.set_type(ignition::msgs::Marker::BOX);
       break;
@@ -1399,6 +1401,19 @@ convert_ign_to_ros(
 
   switch(ign_msg.type())
   {
+#ifdef IGNITION_DOME
+    case ignition::msgs::Marker::ARROW:
+      ros_msg.type = visualization_msgs::Marker::TRIANGLE_LIST;
+      break;
+    case ignition::msgs::Marker::AXIS:
+      ROS_ERROR_STREAM("Unsupported ignition.msgs.marker type " <<
+          "[AXIS]\n");
+      break;
+    case ignition::msgs::Marker::CONE:
+      ROS_ERROR_STREAM("Unsupported ignition.msgs.marker type " <<
+          "[CONE]\n");
+      break;
+#endif
     case ignition::msgs::Marker::NONE:
       ROS_ERROR_STREAM("Unsupported ignition.msgs.marker type " <<
           "[NONE]\n");
@@ -1434,17 +1449,6 @@ convert_ign_to_ros(
     case ignition::msgs::Marker::TRIANGLE_STRIP:
       ROS_ERROR_STREAM("Unsupported ignition.msgs.marker type " <<
           "[TRIANGLE_STRIP]\n");
-      break;
-    case ignition::msgs::Marker::CONE:
-      ROS_ERROR_STREAM("Unsupported ignition.msgs.marker type " <<
-          "[CONE]\n");
-      break;
-    case ignition::msgs::Marker::ARROW:
-      ros_msg.type = visualization_msgs::Marker::TRIANGLE_LIST;
-      break;
-    case ignition::msgs::Marker::AXIS:
-      ROS_ERROR_STREAM("Unsupported ignition.msgs.marker type " <<
-          "[AXIS]\n");
       break;
     default:
       ROS_ERROR_STREAM("Unknown ignition.msgs.marker type " <<

